client_with_ssl.js

```diff
@@ -1,125 +1,126 @@
+
+    const sqlService = await hzClient.getSql();
+    await createMappingForCapitals(sqlService);
+
+    await clearCapitals(sqlService);
+
+    await populateCapitals(sqlService);
+
+    await selectAllCapitals(sqlService);
+
+    await selectCapitalNames(sqlService);
+}
+
+async function createMappingForCapitals(sqlService){
     console.log("Creating a mapping...");
-    // See: https://docs.hazelcast.com/hazelcast/5.0/sql/mapping-to-maps
+    // See: https://docs.hazelcast.com/hazelcast/5.1/sql/mapping-to-maps
     const mappingQuery = `
-        CREATE OR REPLACE MAPPING cities TYPE IMap 
+        CREATE OR REPLACE MAPPING capitals TYPE IMap 
         OPTIONS(
             'keyFormat'='varchar',
             'valueFormat'='varchar'
         );`;
-    await hzClient.getSql().execute(mappingQuery);
+    await sqlService.execute(mappingQuery);
     console.log("The mapping has been created successfully.");
     console.log("--------------------");
+}
 
+async function clearCapitals(sqlService){
     console.log("Deleting data via SQL...");
-    const deleteQuery = "DELETE FROM cities";
-    await hzClient.getSql().execute(deleteQuery);
+    const deleteQuery = "DELETE FROM capitals";
+    await sqlService.execute(deleteQuery);
     console.log("The data has been deleted successfully.");
     console.log("--------------------");
+}
 
+async function populateCapitals(sqlService){
     console.log("Inserting data via SQL...");
     const insertQuery = `
-        INSERT INTO cities
+        INSERT INTO capitals
         VALUES ('Australia', 'Canberra'),
                ('Croatia', 'Zagreb'),
                ('Czech Republic', 'Prague'),
                ('England', 'London'),
                ('Turkey', 'Ankara'),
                ('United States', 'Washington, DC');`;
-    await hzClient.getSql().execute(insertQuery);
+    await sqlService.execute(insertQuery);
     console.log("The data has been inserted successfully.");
     console.log("--------------------");
+}
 
+async function selectAllCapitals(sqlService){
     console.log("Retrieving all the data via SQL...");
-    const sqlResultAll = await hzClient.getSql()
-        .execute("SELECT * FROM cities", [], {returnRawResult: true});
+    const sqlResultAll = await sqlService
+        .execute("SELECT * FROM calpitals", [], {returnRawResult: true});
     for await (const row of sqlResultAll) {
         const country = row.getObject(0);
         const city = row.getObject(1);
         console.log(`${country} - ${city}`);
     }
     console.log("--------------------");
+}
 
+async function selectCapitalNames(sqlService){
     console.log("Retrieving a city name via SQL...");
-    const sqlResultRecord = await hzClient.getSql()
-        .execute("SELECT __key AS country, this AS city FROM cities WHERE __key = ?", ["United States"]);
+    const sqlResultRecord = await sqlService
+        .execute( "SELECT __key, this FROM capitals WHERE __key = ?", "United States");
```

Comment:
This does not work like that as far as I know you need to: 

```suggestion
        .execute( "SELECT __key, this FROM capitals WHERE __key = ?", ["United States"]);
```